### PR TITLE
Jetpack: Revert " Circumvent CRM Wizard behaviour when activating the plugin"

### PR DESCRIPTION
**Pull request by oskosk**
Reverts Automattic/jetpack#22969

Follow-up to #22969

#### Changes proposed in this Pull Request:

* Deals with Jetpack CRM 4.9.1 which updated the behaviour in 1809-gh-Automattic/zero-bs-crm

#### Jetpack product discussion

p1HpG7-eHP-p2#comment-52532

#### Does this pull request change what data or activity we track or use?

no

#### Testing instructions:

1. On the docker environment, with Jetpack connected with the CRM plugin uninstalled
2. Run from the command line `jetpack docker wp option delete zbs_wizard_run`
3. Visit the Jetpack dashbaord
4. Find the CRM card and click  Install CRM
5. Wait until the spinner finishes loading and redirects you to the product cards
6. **Refresh the page immediately**
7. Confirm you don't see the CRM Wizard

### CODE REVIEW

**Review by jeherve — APPROVED** [review]

**Comment by oskosk** [comment]
@jeherve what's the proper thing to do here? Leave the PR removing the changelog or add a changelog file clarifying the revert ?

**Comment by jeherve** [comment]
Since no changelog was created from that changelog entry yet (we didn't release a new version of the Jetpack plugin since that was merged), you do not have anything else to do. Once this gets merged, the changelog entry will be removed and it will never appear anywhere. 👍

**Comment by oskosk** [comment]
Excelente!

**Comment by cleacos** [comment]
We are going to wait until tomorrow morning for the JPCRM release, there is another[ PR we want to include on version 4.9.1]([1810-gh-Automattic/zero-bs-crm](1810-gh-Automattic/zero-bs-crm#pullrequestreview-888998726)).

The major problem is that if JPCRM 4.9.1 is not in the WP.org repo, any test (in the call for the testing process) around the install/active JPCRM will fail. Could it be a problem (temporary) for this call for testing until we release it @jeherve?

**Comment by jeherve** [comment]
> Could it be a problem (temporary) for this call for testing until we release it

As we discussed in Slack, the timing should be okay. 👍

**Pull request merged**
